tests/test_export_rms/test_utils.py

```diff
@@ -31,7 +31,7 @@ def test_get_horizons_in_folder(mock_project_variable):
     # Mock xtgeo.surface_from_roxar to return just the surface name
     with mock.patch(
         "xtgeo.surface_from_roxar",
-        side_effect=lambda _project, name, _category: name,
+        side_effect=lambda _project, name, _category, stype: name,
     ):
         surfaces = get_horizons_in_folder(mock_project_variable, horizon_folder)
```

Comment:
I was hoping to avoid mocking the `xtgeo.surface_from_roxar` here, that should have triggered the bug. 
But I just keeps getting an error related to importing the rmsapi as roxar inside xtgeo. We have mocked this `__version__` in the mocked modules, but somehow it is not picking it up.


> 
> self = <xtgeo.roxutils.roxutils.RoxUtils object at 0x7f733eeef450>, project = <MagicMock id='140132953801680'>, readonly = True
> 
>     def __init__(self, project, readonly=False):
>         self._project = None
>     
>        self._version = roxar.__version__
>       AttributeError: 'NoneType' object has no attribute version
> 
> ../../venv/venv_dataio/lib64/python3.11/site-packages/xtgeo/roxutils/roxutils.py:48: AttributeError

Maybe you know what to do @mferrera ?

Reply:
I can try that in a separate PR :) thanks